sources/packages/backend/workflow/test/test-utils/mock/supporting-user-information-request/check-supporting-user-response-task-mock.ts

```diff
@@ -0,0 +1,28 @@
+import { WorkflowSubprocesses, WorkflowServiceTasks } from "../..";
+import { createMockedWorkerResult } from "../mock.utils";
+
+/**
+ * Creates the mock for 'Check Supporting User Response' completed task.
+ * @param options mock options.
+ * - `supportingUserId` supporting user id to returned the successful check
+ * where the totalIncome is used to unblock the workflow.
+ * - `subprocesses` subprocess reference when the workflow was invoked
+ * using a call activity. It can be defines as one or multiple ones case
+ * the workflow was invoked from subprocess inside another subprocess.
+ * @returns mock for 'Check Supporting User Response' completed task.
+ */
+export function createCheckSupportingUserResponseTaskMock(options: {
+  supportingUserId: number;
```

Comment:
mm, `supportingUserId` is not used in function

Reply:
Done.